src/components/NewChapterForm.tsx

```diff
@@ -0,0 +1,269 @@
+            <div>
+              <div>
+                Do you have any experience in student leadership / club
+                organization / storytelling?{" "}
+              </div>
+              <textarea
+                {...register("leadershipExperience")}
+                className="h-18 w-full resize-y rounded-md px-1 text-black"
+              />
+              <div className="text-sm">
+                <ErrorMessage
+                  errors={errors}
+                  name="leadershipExperience"
+                  render={({ message }) => (
+                    <p>
+                      <span className="text-sunset-orange">* </span>
+                      {message}
+                    </p>
+                  )}
+                />
+              </div>
+            </div>
+            <div className="pt-5">
+              <div>
+                What motivates you to start this initiative in your community?{" "}
+              </div>
+              <textarea
+                {...register("motivation")}
+                className="h-18 w-full resize-y rounded-md px-1 text-black"
+              />
+              <div className="text-sm">
+                <ErrorMessage
+                  errors={errors}
+                  name="motivation"
+                  render={({ message }) => (
+                    <p>
+                      <span className="text-sunset-orange">* </span>
+                      {message}
+                    </p>
+                  )}
+                />
+              </div>
+            </div>
+            <div className="pt-5">
+              <div>
+                Please list three 1 hour time blocks with your avalibility in
+                the next week{" "}
+              </div>
+              <input
+                {...register("availabilities")}
+                className="h-8 w-full rounded-md px-1 text-black"
+                placeholder="Include the date (mm-dd-yyyy), time (hh:mm am/pm), and your timezone"
+              />
+              <div className="text-sm">
+                <ErrorMessage
+                  errors={errors}
+                  name="availabilities"
+                  render={({ message }) => (
+                    <p>
+                      <span className="text-sunset-orange">* </span>
+                      {message}
+                    </p>
+                  )}
+                />
+              </div>
+            </div>
+            <div className="pt-5">
+              <div>What questions do you have for us? </div>
+              <textarea
+                {...register("questions")}
+                className="h-8 w-full resize-y rounded-md px-1 text-black"
+              />
+            </div>
+          </div>
+          <div className="grid place-items-center pt-5">
+            <div className="w-1/12 cursor-pointer rounded-lg bg-white px-1 py-1 text-center ">
+              <input type="submit" className="items-center text-dark-teal" />
```

Comment:
I think we can remove line 260 and 262 and move all the stylings into `<input ... />`. Something like this

```jsx
<input
      type="submit"
      className="w-1/12 cursor-pointer items-center rounded-lg bg-white px-1 py-1 text-dark-teal"
/>
```